mediajson/encode.py

```diff
@@ -75,8 +74,6 @@ def encode_value(o: MediaJSONSerialisable,
     elif isinstance(o, uuid.UUID):
         return str(o)
     elif isinstance(o, Timestamp):
-        return o.to_tai_sec_nsec()
-    elif isinstance(o, TimeOffset):
         return o.to_sec_nsec()
```

Comment:
Is this an intended change in behaviour? Stopping this returning tai. Same elsewhere

Reply:
There is no difference between `to_tai_sec_nsec()` and `to_sec_nsec()` - the former calls the latter. I can put it back if you want but we tend to use `to_sec_nsec()` or `str(timestamp)` in the Cloudfit code and I think the `to_tai...` methods should be deprecated as they give the wrong impression of what is actually implemented.

Reply:
Fair enough. Probably fine as is then :+1: